Userland/Utilities/top.cpp

```diff
@@ -19,6 +20,23 @@
 #include <sys/ioctl.h>
 #include <unistd.h>
 
+struct TopOption {
+    enum class SortBy {
+        pid = 0,
+        tid,
+        priority,
+        username,
+        state,
+        virt,
+        phys,
+        cpu,
+        name
```

Comment:
Please follow [project coding style](https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md) for enum values:

> Enum members should use InterCaps with an initial capital letter.

Reply:
Oh, stupid mistake.. thanks!